src/middleware/queries/authorizations.ts

```diff
@@ -59,7 +59,6 @@ export function authorizationsQuery(
 ): QueryOptions<PaginatedQueryArgs<QueryArgs<Authorization, AuthorizationArgs>>> {
   const { args, filter } = createAuthorizationFilters(filters);
 
-  const idField = paddedIds ? 'id' : 'authId: id';
   const orderBy = paddedIds
     ? `${AuthorizationsOrderBy.CreatedBlockIdAsc}`
```

Comment:
order by CREATED_EVENT_ID_ASC this ensure correct ordering even if multiple authorizations are emitted in a single block
```suggestion
    ? `${AuthorizationsOrderBy.CreatedEventIdAsc}`
```